data_model/src/transaction.rs

```diff
@@ -67,14 +67,18 @@ pub trait Txn {
                 let instruction_count: usize = instructions.iter().map(Instruction::len).sum();
 
                 if instruction_count as u64 > limits.max_instruction_number {
-                    return Err(TransactionLimitError(String::from(
-                        "Too many instructions in payload",
+                    return Err(TransactionLimitError(format!(
+                        "Too many instructions in payload, max number is {}",
```

Comment:
```suggestion
                        "Too many instructions in payload, max number is {}. Got {}",
```

Reply:
I perfer not to use dots in error messages so that error message chain looks better. I'll use comma here